src/categorical_algebra/DataMigrations.jl

```diff
@@ -335,9 +340,32 @@ function (M::SigmaMigrationFunctor)(d::ACSet; n=100)
       res[domval,h] = codomval
     end
   end
+  #Go back and make sure attributes that ought to have 
+  #specific values because of d do have those values.
+  for (k,kdom,kcod) in attrs(S)
+    f = hom_map(F,k)
+    #split f into its hom part and its attr part
+    f1,f2 = split_r(f)
+    f1 = hom_map(i2,f1)#,hom_map(i2,f2)
+    for i in parts(d,kdom)
+      oldval = subpart(d,i,k)
+      #Find where i goes under alpha, and then where that goes
+      #under the hom part of f, by walking the spans in rel_res.
+      j = rel_res[only(incident(rel_res,i,Symbol("src_α_$kdom"))),
```

Comment:
use `src_a, tgt_a = add_srctgt(Symbol("α_$kdom"))` to abstract away the string hacking (and have one place to make a change if we want to change the convention)

(also below with `"src_$f1"`)